### PR TITLE
[Stats Refresh] Update card header

### LINKED ISSUE #10498: [Stats Refresh] Update card header
Change the card heading from the original design to the new:

card_header_change

### PR DESCRIPTION
**Pull request by ScoutHarris**
Fixes #10498 

This actually encompasses these changes:
1) Updates the card header to the new design.
2) Removes the full border on the cards, and now has only a top and bottom.
3) Updates margins for the Latest Post Summary text.
4) Fixes an issue where All Time Stats rows were being duplicated on pull to refresh.

The new card header should match this Zeplin design zpl://screen?pid=5a86dd8dfaff0fa0290982db&sid=5bf2a8d9e6b0427eb9a22108 (Full Stats Refresh > Heading Bar Changes).

To test:

---
On a site with Latest Post Summary data, Go to Stats > Insights. Verify the view is:

with_lps

---
On a site with _no_ Latest Post Summary data, Go to Stats > Insights. Verify the view is:

no_lps

---
On a site with _no_ All Time Stats data, Go to Stats > Insights. Verify the view is:

no_ats

### CODE REVIEW

**Comment by ScoutHarris** [comment]
@SylvesterWilmott since the header ellipses are disabled for now, here's a screenshot of how it will look:

with_ellipsis

**Comment by jklausa** [comment]
>Fixes an issue where All Time Stats rows were being duplicated on pull to refresh.

boo :(

File: WordPress/Classes/ViewRelated/Stats/Insights/SimpleTotalsCell.swift [hunk 1]
```diff
@@ -1,32 +1,31 @@
 import UIKit
 
-class SimpleTotalsCell: UITableViewCell {
+class SimpleTotalsCell: UITableViewCell, NibLoadable {
```
Comment (jklausa) [inline WordPress/Classes/ViewRelated/Stats/Insights/SimpleTotalsCell.swift]: I love seeing `NibLoadable` spread!

**Comment by SylvesterWilmott** [comment]
Hey thanks!

Can we reduce the height of them a touch, It appears we have them set to 40pt, can we go for 36pt?

File: WordPress/Classes/ViewRelated/Stats/Insights/SimpleTotalsCell.swift (hunk 1 above)
Comment (ScoutHarris) [reply WordPress/Classes/ViewRelated/Stats/Insights/SimpleTotalsCell.swift]: I'm loving it too! It's incredibly handy.

**Comment by ScoutHarris** [comment]
@SylvesterWilmott - OK, I think I got it this time. Thanks for noticing. How's this?

header_margins

**Comment by SylvesterWilmott** [comment]
Perfect @ScoutHarris thanks!

**Pull request merged**